### ISSUE TITLE
AnnotationSegment class is not exported

### ISSUE THREAD

**Issue by arazinger** [google/charts#239]
I want to mix range an line annotations.
When setting up annotations "manually" one can do the following:

    behaviors: [
        charts.RangeAnnotation([
            charts.RangeAnnotationSegment(...),
            charts.LineAnnotationSegment(...),
        ]),
    ],

But if we want to make it in a more flexible way:

    List<charts.AnnotationSegment> _annotationSegments;
    _annotationSegments.add(...);
    ...
    behaviors: [
        charts.RangeAnnotation(_annotationSegments),
    ],

... we have a problem because base AnnotationSegment is not exported.

Is there any possibility to export AnnotationSegment class?

**Comment by handersonc** [comment]
I also need that. I'm getting a problem but I'm not sure is this is going to solve that.

The problem is that when you add RangeAnnotationSegment and LineAnnotationSegment
as a behaviors it seems the chart takes the first type. So, if we add

List<charts.LineAnnotationSegment<dynamic>> lineAnotations = [];
List<charts.RangeAnnotationSegment<dynamic>> rangeAnotations = [];

behaviors: [
    charts.RangeAnnotation(lineAnotations),
    charts.RangeAnnotation(rangeAnotations),
]

in the previous case the chart only display the LineAnnotationSegment if we change the order
as next only display RangeAnnotationSegment

behaviors: [
   charts.RangeAnnotation(rangeAnotations),
    charts.RangeAnnotation(lineAnotations)
]

**Comment by reducedcarpet** [comment]
Yes, I'm also getting this issue. Can't iteratively make a mixture of Line and Range annotations.
